webapp/src/ts/components/filters/status-filter/status-filter.component.html

```diff
@@ -1,36 +1,13 @@
 <div class="mm-status-filter" [id]="fieldId">
-  <multi-dropdown-filter
-    *ngIf="!inline"
-    [items]="allStatuses"
-    (applyFilter)="applyFilter($event)"
-    [label]="getFilterLabel"
-    [disabled]="disabled"
-  >
-    <span class="fa fa-circle-o"></span>
-    <ul class="filter-options">
-      <ng-container
-        [ngTemplateOutlet]="optionsTemplate"
-        [ngTemplateOutletContext]="{ statuses: statuses, filter: dropdownFilter }">
-      </ng-container>
-    </ul>
-  </multi-dropdown-filter>
+  <ul class="filter-options">
+    <li role="presentation" class="dropdown-header group-header">{{ 'status.review.title' | translate }}</li>
+    <li role="presentation" *ngFor="let status of statuses.verified" [class.selected]="filter?.selected.has(status)" [class.disabled]="disabled">
+      <a role="menuitem" tabindex="-1" [attr.data-value]="status" (click)="!disabled && filter?.toggle(status)">{{ 'status.review.' + status | translate }}</a>
```

Comment:
```suggestion
   <li
      role="presentation"
      *ngFor="let status of statuses.verified"
      [class.selected]="filter?.selected.has(status)" [class.disabled]="disabled">
      <a
        role="menuitem"
        tabindex="-1"
        [attr.data-value]="status"
        (click)="!disabled && filter?.toggle(status)">
        {{ 'status.review.' + status | translate }}
      </a>
```

Reply:
Same for the `li` on L8 below to improve readability

Reply:
Fixed!